src/app/components/top/solicitudes/nuevaSolicitud.component.ts

```diff
@@ -114,77 +115,57 @@ export class NuevaSolicitudComponent implements OnInit {
     }
 
-    searchEnd(resultado) {
-        this.loading = false;
-        if (resultado.err) {
-            this.plex.info('danger', resultado.err);
-            return;
-        }
-        this.resultadoBusqueda = resultado.pacientes;
-    }
-
-    onSearchClear() {
-        this.resultadoBusqueda = [];
-        this.paciente = null;
-    }
-
-    // ----------------------------------
-
     // Componente paciente-listado
 
     seleccionarPaciente(paciente): void {
-        this.resultadoBusqueda = [];
-        this.showSeleccionarPaciente = false;
-        if (paciente && paciente.id) {
-            this.paciente = paciente;
-        } else {
-            this.plex.info('warning', 'Paciente no encontrado', '¡Error!');
-        }
-
-        setTimeout(() => {
-            this.wizardActivo = true;
-            let promise = this.plex.wizard({
-                id: 'top:fechaSolicitud',
-                updatedOn: moment('2019-03-08').toDate(),
-                steps: [
-                    {
-                        title: 'Fecha de solicitud',
-                        content: 'Recuerde registrar en este campo la fecha en que el profesional solicita la prestación y no la fecha en que se registra en el sistema'
-                    },
-                ],
-                forceShow: false,
-                fullScreen: false,
-                showNumbers: false
-            });
-
-            // Devuelve una promise sólo si se mostró el wizard
-            if (promise) {
-                promise.then(() => this.wizardActivo = false);
-            } else {
-                this.wizardActivo = false;
+        this.pacienteService.getById(paciente).subscribe(
+            resultado => {
+                this.paciente = resultado;
+                setTimeout(() => {
+                    this.wizardActivo = true;
+                    let promise = this.plex.wizard({
+                        id: 'top:fechaSolicitud',
+                        updatedOn: moment('2019-03-08').toDate(),
+                        steps: [
+                            {
+                                title: 'Fecha de solicitud',
+                                content: 'Recuerde registrar en este campo la fecha en que el profesional solicita la prestación y no la fecha en que se registra en el sistema'
+                            },
+                        ],
+                        forceShow: true,
+                        fullScreen: false,
+                        showNumbers: false
+                    });
+                    // Devuelve una promise sólo si se mostró el wizard
+                    if (promise) {
+                        promise.then(() => this.wizardActivo = false);
+                    } else {
+                        this.wizardActivo = false;
+                    }
+                }, 1000);
+            },
+            error => {
+                this.plex.info('warning', 'Intente nuevamente', 'Error en la búsqueda de paciente');
```

Comment:
```suggestion
                this.plex.info('danger', 'Intente nuevamente', 'Error en la búsqueda de paciente');
```

Acá hay que volver a la pantalla anterior, alguien hizo trampa y cambio la URL. O se rompio MPI xd

Reply:
done